hera_cal/reflections.py

```diff
@@ -267,25 +228,439 @@ def write_reflections(self, output_calfits, input_calfits=None, overwrite=False,
             if cal.Ntimes > self.Ntimes:
                 time_array = cal.time_array
             else:
-                time_array = self.reflection_times
+                time_array = self.times
             if cal.Nfreqs > self.Nfreqs:
                 freq_array = cal.freq_array
             else:
                 freq_array = self.freqs
             kwargs = dict([(k, getattr(cal, k)) for k in ['gain_convention', 'x_orientation',
                                                           'telescope_name', 'cal_style']])
         else:
-            time_array = self.reflection_times
+            time_array = self.times
             freq_array = self.freqs
             kwargs = {}
 
-        uvc = io.write_cal(output_calfits, rgains, freq_array, time_array, flags=flags, quality=quals, 
-                           total_qual=tquals, outdir=os.path.dirname(output_calfits), zero_check=False, 
-                           overwrite=overwrite, history=version.history_string(add_to_history), **kwargs)
+        echo("...writing {}".format(output_calfits))
```

Comment:
Shouldn't we be passing in a verbose option? Or do you really want this to always print?